card/lib/card/cache.rb

```diff
@@ -25,7 +26,7 @@ class Cache
     class << self
       def [] klass
         raise "nil klass" if klass.nil?
-        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Rails.cache)
+        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Cardio.cache)
```

Comment:
just webrick is fine.  when I first looked at card-gem the browser failed on the cache before it loaded the first page.